WordPress/openverse-frontend
# Add small dropdown button variant

## Fixes


Fixes #510 by @dhruvkb 

## Description


Adds a small size dropdown button option in the most ad-hoc way possible... but it does work for now. In the future it'd be nice to refactor it top-down to use our VButton and other standard components, but it's not possible to do in the amount of time we have right now.

## Testing Instructions

Checkout this branch and run `pnpm storybook` and visit the [dropdown button story](http://localhost:6006/?path=/story/components-dropdownbutton--default) and toggle between the medium and small size props.

Also check out the [small audio track component story](http://localhost:6006/?path=/story/components-audio-track--full-small) and verify that the download button is the same height as the play/pause button.

## Checklist

- [N/A] I added or updated tests for the changes I made (if applicable).

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
